### DISCUSSION TITLE
How do I add other federated feeds to my new mastodon server

### DISCUSSION THREAD

**Discussion by kmages** [mastodon/mastodon#20425] [General]
How do I get all federated toots to appear in my new server (ver.news) federated feeds?

**Comment by AlanBell** [comment]
you might want to add an activitypub relay to your instance to get some more stuff on the federated timeline of your server

**Comment by kmages** [comment]
Sorry to be an idiot. How do I do this?kgmSpelling and grammar curiosities courtesy of my mobile phone.KenMages.comOn Nov 11, 2022, at 2:51 PM, AlanBell ***@***.***> wrote:﻿
you might want to add an activitypub relay to your instance to get some more stuff on the federated timeline of your server
